corehq/apps/export/views.py

```diff
@@ -1703,16 +1707,53 @@ def _check_export_size(self, export_instances, filters):
 
 class DownloadNewFormExportView(GenericDownloadNewExportMixin, DownloadFormExportView):
     urlname = 'new_export_download_forms'
-    filter_form_class = FilterFormESExportDownloadForm
+    filter_form_class = EmwfFilterFormExport
+
+    @use_daterangepicker
+    @use_select2
+    @use_angular_js
+    @location_safe
+    @method_decorator(login_and_domain_required)
+    def dispatch(self, request, *args, **kwargs):
+        if not (self.has_edit_permissions
+                or self.has_view_permissions
+                or self.has_deid_view_permissions):
+            raise Http404()
+        return super(DownloadNewFormExportView, self).dispatch(request, *args, **kwargs)
 
     def _get_export(self, domain, export_id):
         return FormExportInstance.get(export_id)
 
-    def get_filters(self, filter_form_data):
+    def get_filters(self, filter_form_data, mobile_user_and_group_slugs):
         filter_form = self._get_filter_form(filter_form_data)
-        form_filters = filter_form.get_form_filter()
+        form_filters = filter_form.get_form_filter(mobile_user_and_group_slugs)
         return form_filters
 
+    @property
+    def page_context(self):
+        parent_context = super(DownloadNewFormExportView, self).page_context
+        parent_context['new_export_filters'] = LocationRestrictedMobileWorkerFilter(
+            self.request, self.request.domain).render()
+        return parent_context
+
+    def _process_filters_and_specs(self, in_data):
+        """Returns a the export filters and a list of JSON export specs
+        """
+        filter_form_data, export_specs = self._get_form_data_and_specs(in_data)
+        # tried self.request.POST.getlist(EMWF.slug)
+        import re
+        regex = re.compile('(emw=){1}([^&]*)(&){0,1}')
+        matches = regex.findall(filter_form_data['emw'])
+        mobile_user_and_group_slugs = [n[1] for n in matches]
+        try:
+            export_filter = self.get_filters(filter_form_data, mobile_user_and_group_slugs)
+        except ExportFormValidationException:
+            raise ExportAsyncException(
+                _("Form did not validate.")
+            )
+
+        return export_filter, export_specs
+
 
 class BulkDownloadNewFormExportView(DownloadNewFormExportView):
```

Comment:
No changes to BulkDownloadNewFormExportView. The changes below are for DownloadNewCaseExportView. Diff misguided you :)